nebula-query-and-search/main/classes/SOQL.cls-meta.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <ApexClass xmlns="http://soap.sforce.com/2006/04/metadata">
-    <apiVersion>57.0</apiVersion>
+    <apiVersion>58.0</apiVersion>
```

Comment:
crucial API version upgrade here - otherwise the test will continue to fail. I didn't upgrade the _other_ API versions so as to not distract from this particular change

Reply:
Makes sense, but I say let's go ahead and bump the API version everywhere!